partiql-logical-planner/src/graph.rs

```diff
@@ -250,10 +184,12 @@ impl GraphToLogical {
         &mut self,
         pattern: &ast::GraphPattern,
     ) -> Result<Vec<MatchElement>, String> {
-        if pattern.mode.is_some() {
+        if pattern.match_mode.is_some() {
+            // The current graph engine is basically doing RepeatableElements currently
```

Comment:
Could you clarify this comment? What is `RepeatableElements`?

Reply:
The current graph engine is basically executing as though `pattern.match_mode` is `Some(GraphMatchMode::RepeatableElements)` currently